### PR TITLE
Add info about surface density from CO2STORE model

### PR DESCRIPTION
**Pull request by totto82**
This outputs the reference density used by the CO2STORE model.

### CODE REVIEW

**Comment by totto82** [comment]
jenkins build this please

**Comment by bska** [comment]
jenkins build this please

File: src/opm/material/fluidsystems/blackoilpvt/Co2GasPvt.cpp [hunk 1]
```diff
@@ -39,15 +39,9 @@ initFromState(const EclipseState& eclState, const Schedule&)
 
     setEnableVaporizationWater(eclState.getSimulationConfig().hasVAPOIL() || eclState.getSimulationConfig().hasVAPWAT());
 
-    if (!eclState.getTableManager().getDensityTable().empty()) {
-        OpmLog::warning("CO2STOR is enabled but DENSITY is in the deck. \n"
-                        "The surface density is computed based on CO2-BRINE "
-                        "PVT at standard conditions (STCOND) and DENSITY is ignored");
-    }
```
Comment (bska) [inline src/opm/material/fluidsystems/blackoilpvt/Co2GasPvt.cpp:46]: Does the use no longer need this information?

File: src/opm/material/fluidsystems/blackoilpvt/BrineCo2Pvt.cpp [hunk 2]
```diff
@@ -71,6 +71,13 @@ initFromState(const EclipseState& eclState, const Schedule&)
 
     brineReferenceDensity_[regionIdx] = Brine::liquidDensity(T_ref, P_ref, salinity_[regionIdx], extrapolate);
     co2ReferenceDensity_[regionIdx] = CO2::gasDensity(T_ref, P_ref, extrapolate);
+
+    OpmLog::info("CO2STORE is enabled. \n The surface density of CO2 is  " + std::to_string(co2ReferenceDensity_[regionIdx])
+                 + "kg/m3 \n The surface density of Brine is  " + std::to_string(brineReferenceDensity_[regionIdx])
+                 + "kg/m3"
+                 + "\n The surface densities are computed using the reference pressure ( " + std::to_string(P_ref) 
+                 + "Pa) and the reference temperature (" +  std::to_string(T_ref) + "K)."
```
Comment (bska) [inline src/opm/material/fluidsystems/blackoilpvt/BrineCo2Pvt.cpp:79]: Do we not anticipate anyone running `CO2STORE` cases in unit conventions other than METRIC?

**Review by bska — COMMENTED** [review]
This is fine of course, but I wonder why you removed the warning concerning the `DENSITY` keyword.

File: src/opm/material/fluidsystems/blackoilpvt/Co2GasPvt.cpp (hunk 1 above)
Comment (totto82) [reply src/opm/material/fluidsystems/blackoilpvt/Co2GasPvt.cpp:46]: It is duplicated in BrineCo2Pvt.cpp. So now the user only get the information once.

Comment (bska) [reply src/opm/material/fluidsystems/blackoilpvt/Co2GasPvt.cpp:46]: > It is duplicated in BrineCo2Pvt.cpp. So now the user only get the information once.

Okay, I see.

File: src/opm/material/fluidsystems/blackoilpvt/BrineCo2Pvt.cpp (hunk 2 above)
Comment (bska) [reply src/opm/material/fluidsystems/blackoilpvt/BrineCo2Pvt.cpp:79]: > For simplicity I just dumped everything in SI units. I don't think the need is urgent, but it shouldn't be too hard to convert using the `from_si` functions.

Right.  You'll probably need the [`name()`](https://github.com/OPM/opm-common/blob/980ac2599acf475129019e8de8a352b05d8b280d/opm/input/eclipse/Units/UnitSystem.hpp#L125) function to retrieve the associate unit strings as well, but that's easy enough.

**Review by bska — APPROVED** [review]
Thanks a lot for outlining the reasoning here.  We'll do unit conversion in follow-up work so I'll merge into master.

**Pull request merged**